### DISCUSSION TITLE
Packaging/box/dashboard

### DISCUSSION THREAD

**Discussion by sommo** [micbay/RaceTimerAndController#17] [General]
Hi all,
Have you any idea how to set all the hw to fit the slot car track?
I'm thinking to use a plexiglass and do a sort of bridge and some hole to fit display, ledbar... I will post some hand drawing soon

**Comment by micbay** [comment]
Hi @sommo,
I'm not sure how you originally found this project, but if you didn't follow a link from a post on [Slot Car Forum](https://www.slotforum.com/forums/), then I would point you to that website/community and suggest also posting such questions there. They are a world wide, very responsive, well versed, community in all things slot car, in particular track and car builds and customizations. In fact it, they may even have some existing posts of solutions that fit what you are looking for.

**Comment by sommo** [comment]
i will take a look, thank you!
